bin/v2.js

```diff
@@ -525,8 +555,111 @@ const releaseCmd = (cmdObj, bucketNames, lambdaFunctions, environments, bucketPa
     .catch(err => console.error(err));
 }
 
+const cleanupCmd = async (cmdObj, bucketNames, lambdaFunctions, environments, bucketPath, packageName) => {
+
```

Comment:
I'd prefer no newline, but I don't do much JS at all, so take that with a grain of salt

Reply:
```suggestion
```